src/lib/default.nix

```diff
@@ -287,7 +293,10 @@
     sanitizedRelPath = l.removePrefix "/" (l.toString (l.toPath "/${path}"));
   in
     if absolute
-    then "/${sanitizedRelPath}"
+    then
+      l.path {
+        path = "/${sanitizedRelPath}";
+      }
```

Comment:
I think there is no need to use `builtins.path` in sanitizePaths. sanitizePaths is just a string operation that normalizes `//` to `/` etc.